storm/datadog_checks/storm/data/conf.yaml.example

```diff
@@ -0,0 +1,25 @@
+init_config:
```

Comment:
This is not the proper format

Reply:
Yeah, I hadn't taken the time to convert the conf.yaml file to the newer format yet. Was mostly focusing on structure/test changes for this PR.

Reply:
Ok. Let's do this still since we are at it